tests/profile/tracking/test_tracking_sessions.py

```diff
@@ -0,0 +1,280 @@
+import os
+import json
+import pytest
+from uuid import uuid1, uuid5
+
+import kano_profile.tracker.tracking_sessions as tracking_sessions
+from kano_profile.tracker.tracking_session import TrackingSession
+from kano_profile.paths import tracker_dir, PAUSED_SESSIONS_FILE, \
+    tracker_events_file
+
+
+'''
+{"name": "kano-dashboard", "started": 1459774858, "pid": 9999, "elapsed": 7, "finished": true, "app_session_id": "d1ea40d1-ddc8-5a62-97a2-980b71a9bc19"}
+'''
+
+def format_session(name, started, pid, elapsed, finished):
+    return {
+        "name": name,
+        "started": started,
+        "pid": pid,
+        "elapsed": elapsed,
+        "finished": finished,
+        "app_session_id": str(uuid5(uuid1(), '{}{}'.format(name, pid)))
+    }
+
+
+def setup_session_fixtures(directory, sessions):
```

Comment:
These could also be abstracted to pytest fixtures and used a bit cleaner here (general note).